third_party/blink/renderer/core/layout/layout_object.cc

```diff
@@ -419,12 +419,10 @@ LayoutObject::~LayoutObject() {
 
   // If recording/replaying and in a nondeterministic execution, allow
   // style_ to leak, since it may otherwise get destroyed in a
-  // non-desterministic fashion and remove itself from font-fallback-maps
+  // non-deterministic fashion and remove itself from font-fallback-maps
   // that are accessed deterministically.
   // See https://linear.app/replay/issue/RUN-1758/fontfallbackmap-items-getting-removed-non-deterministically
-  if (recordreplay::IsRecordingOrReplaying("leak-references") &&
-      recordreplay::AreEventsDisallowed())
-  {
+  if (recordreplay::AreEventsDisallowed("~LayoutObject")) {
```

Comment:
This is one of 4 places where `"leak-references"` was removed - Is that intentional?

Reply:
Yes, testing both IsRecordingOrReplaying and AreEventsDisallowed is redundant.  The AreEventsDisallowed call has its own feature test.